WS2012R2/lisa/xml/NET_Tests.xml

```diff
@@ -571,6 +572,30 @@ permissions and limitations under the License.
                 <param>Switch_Name=external</param>
             </testParams>
         </test>
+
+        <test>
+            <testName>Multicast</testName>
+            <setupScript>
+                <file>setupscripts\RevertSnapshot.ps1</file>
+                <file>setupscripts\NET_ADD_NIC_MAC.ps1</file>
+                <file>setupscripts\StartVM.ps1</file>
+            </setupScript>
+            <pretest>setupscripts\NET_SendIPtoVM.ps1</pretest>
+            <testScript>NET_Multicast.sh</testScript>
+            <files>remote-scripts/ica/utils.sh,remote-scripts/ica/NET_set_static_ip.sh,
+                remote-scripts/ica/NET_Multicast.sh</files>
+            <testparams>
+                <param>NIC=NetworkAdapter,Private,Private</param>
+                <param>TC_COVERED=NET-10</param>
+                <param>STATIC_IP=10.10.10.10</param>
```

Comment:
the private IPs STATIC_IP and STATIC_IP2 are used in at least 2 other network tests.
Please assign other IPs not defined in any other tests.